sdk/identity/Azure.Identity/src/MsalPublicClient.cs

```diff
@@ -169,6 +170,18 @@ protected virtual async ValueTask<AuthenticationResult> AcquireTokenInteractiveC
             {
                 builder.WithAuthority(AuthorityHost.AbsoluteUri, tenantId);
             }
+            if (useEmbeddedWebView != null)
+            {
+                builder.WithUseEmbeddedWebView(useEmbeddedWebView == true);
+            }
+            if (systemWebViewOptions != null)
+            {
+                builder.WithSystemWebViewOptions(systemWebViewOptions);
+            }
+            if (embeddedWebViewOptions != null)
+            {
+                builder.WithEmbeddedWebViewOptions(embeddedWebViewOptions);
+            }
```

Comment:
We need tests to validate that these properties are set or not set as expected based on the credential options.

Reply:
> We need tests to validate that these properties are set or not set as expected based on the credential options.

@christothes Could you provide an test case that I can take it as an example? Thanks

Reply:
> We need tests to validate that these properties are set or not set as expected based on the credential options.

@christothes Could you provide an test case that I can take it as an example? Thanks

Reply:
Sure - take a look at tests that use the `MockMsalPublicClient`, like [this](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/tests/InteractiveBrowserCredentialTests.cs#L185)

Reply:
Thanks @christothes I have add 2 test cases please take a look.